### ISSUE TITLE
Any feedback?

### ISSUE THREAD

**Issue by evert** [Sanshain/test_changelog#1]
Github was being creepy and said you tried my [changelog-tool](https://github.com/evert/changelog-tool) package. Anything you would want from it that it doesn't do yet? Even if you don't want to use it, it's helpful to know!

**Comment by Sanshain** [comment]
It turned out a curious story, which is apparently related to algorithms by which the github service determines the packages usages in another packages. 

The thing is, I didn't use the package. The other day I started working on my own package for automatically generating the changelog file using hook pre-push as a trigger for regeneration. 

When I choose the package name, I've entered in the search on npm keyword [changelog](https://www.npmjs.com/search?q=changelog) to check which names are occupied. Not finding anything similar in the top twenty, I decided to use the `changelog-tool`, but did not check for an exact match. Then I needed a repository to test my package (this is what it is) and I installed my package into it through github for a while. Therefore, in `devDependencies` in `package.json` featured the corresponding name (`changelog-tool`). Eventually, when I was ready to publish my package on npm, I discovered that a package named `changelog-tool` already exists, and then I changed the name to `npm-changelog` everywhere. But github, apparently, has already managed to index that I have a package with the name `changelog-tool` in the dependencies and associated it with package `changelog-tool`.  Here's a funny situation

I looked at your package after that. It seemed interesting, but it requires (as far as I understand) separate maintenance of commits and changelog (the exception is the use of the `release` option). And this is a little not what I needed. But despite this, thank you for contacting me. I think taking care of users and sharing experiences is always a good thing.
